examples/quick_start/anthropic_example_complete.py

```diff
@@ -19,7 +19,7 @@ def few_shot_qa(s, question):
 \n\nAssistant: Rome
 """)
     s += "\n\nHuman: " + question + "\n"
-    s += "\n\nAssistant:" + sgl.gen("answer", stop="\n", temperature=0)
+    s += "\n\nAssistant:" + sgl.gen("answer", stop="\\n", temperature=0)
```

Comment:
is "\\n" a typo?

Reply:
No, the new API apparently does not take empty stop sequences, and considers '\n' empty....